pkgs/applications/editors/textadept/deps.nix

```diff
@@ -36,7 +36,7 @@
   };
   # cdk
   "t20200923.tar.gz" = {
-    url = "http://github.com/ThomasDickey/cdk-snapshots/archive/refs/tags/t20200923.tar.gz";
```

Comment:
This file is likely generated,so this will be reverted.

Reply:
Thanks for the review. I reverted this change.